solidity/dashboard/src/components/LiquidityRewardCard.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react"
+import DoubleIcon from "./DoubleIcon"
+import * as Icons from "./Icons"
+import { SubmitButton } from "./Button"
+import Card from "./Card"
+
+const LiquidityRewardCard = ({ title, MainIcon, SecondaryIcon, viewPoolLink }) => {
+  return (
+    <Card className={"tile"}>
+      <div className={"liquidity__card-title-section"}>
+        <DoubleIcon
+          MainIcon={MainIcon}
+          SecondaryIcon={SecondaryIcon}
+          className={`liquidity__double-icon-container`}
+        />
+        <h2 className={"h2--alt text-grey-70"}>{title}</h2>
+      </div>
+      <div className={"liquidity-card-subtitle-section"}>
+        <span className="text-grey-40">Uniswap Pool&nbsp;</span>
+        <a
+          href={viewPoolLink}
+          className="arrow-link text-small"
+          style={{ marginLeft: "auto", marginRight: "2rem" }}
```

Comment:
We can drop this `style` prop.

Reply:
I think this was fixed by you in one of your commits.